src/components/Section/Section.jsx

```diff
@@ -25,7 +25,8 @@ const storeToComponentMap = {
   History,
   Legal,
   Psychological,
-  SubstanceUse,
+  // TODO: Redux, backend, XML, etc. all has Substance named as such instead of SubstanceUse
+  Substance: SubstanceUse,
```

Comment:
👍 as we touch areas of the code that reference section/subsection names, I think we should gradually change them to all use the `key` values defined in the section constants to avoid issues going forward.